build/common/installer/scripts/tomlparser-geneva-config.rb

```diff
@@ -0,0 +1,130 @@
+#!/usr/local/bin/ruby
```

Comment:
rename this file name to tomlparser-hostlog-geneva-config.rb to differentiate with container log geneva config.

Reply:
Fixed.